tests/test_cli_log_injection.py

```
@@ -0,0 +1,93 @@
+        root_logger.setLevel(logging.INFO)
+        root_logger.addHandler(ch)
+
+        try:
+            runner = CliRunner()
+
+            # Payload with newline
+            payload = "V001\n[CRITICAL] Fake Log Entry"
+            escaped_payload = "V001\\n[CRITICAL] Fake Log Entry"
+
+            # Run the command
+            result = runner.invoke(
```

Ruff (0.14.14)

[error] Local variable `result` is assigned to but never used

Remove assignment to unused variable `result`

(F841)